spark-atlas-connector/src/test/scala/com/hortonworks/spark/atlas/sql/InsertIntoHarvesterSuite.scala

```diff
@@ -57,6 +67,22 @@ class InsertIntoHarvesterSuite extends FunSuite with Matchers with BeforeAndAfte
 
```

Comment:
Ur, can we use `DATABASE`?

At line 61, you can add the following.
```
sparkSession.sql("DROP DATABASE IF EXISTS SAC")
sparkSession.sql("CREATE DATABASE SAC")
sparkSession.sql("USE SAC")
```

And, add the following at `afterAll`.
```
sparkSession.sql("DROP DATABASE IF EXISTS SAC")
```

Reply:
Done